python-package/lets_plot/plot/sampling.py

```diff
@@ -15,10 +15,78 @@
 
 
 def sampling_random(n, seed=None):
+    """
+    Return a subset of randomly selected items.
+
+    Parameters
+    ----------
+    n : int
+        Number of items to return.
+    seed : int
+        Number used to initialize a pseudo random number generator.
+
+    Returns
+    -------
+    Series or `DataFrame`
+        A new object of same type as caller containing n items randomly sampled from the caller object.
+
+    Examples
+    --------
+    .. jupyter-execute::
+        :linenos:
+        :emphasize-lines: 10
+
+        import numpy as np
+        from lets_plot import *
+        LetsPlot.setup_html()
+        np.random.seed(27)
+        n = 15000
+        x, y = np.random.multivariate_normal(mean=[0,0],
+                                             cov=[[.9, -.6], [-.6, .9]],
+                                             size=n).T
+        ggplot({'x': x, 'y': y}, aes(x='x', y='y')) + \\
+            geom_point(sampling=sampling_random(1000, 35))
+
+    """
     return _sampling('random', n=n, seed=seed)
 
 
 def sampling_random_stratified(n, seed=None, min_subsample=None):
+    """
+    Randomly sample from each stratum (subgroup).
+
+    Parameters
+    ----------
+    n : int
+        Number of items to return.
+    seed : int
+        Number used to initialize a pseudo random number generator.
+    min_subsample: int
+        Minimal number of items in sub sample.
+
+    Returns
+    -------
+    Series or `DataFrame`
+        A new object of same type as caller containing n items sampled.
+
+    Examples
+    --------
+    .. jupyter-execute::
+        :linenos:
+        :emphasize-lines: 10
+
+        import numpy as np
+        from lets_plot import *
+        LetsPlot.setup_html()
+        np.random.seed(27)
+        data = dict(
+            x = np.random.normal(0, 1, 1100),
+            y = np.random.normal(0, 1, 1100),
+            cond = ['1' for i in range(100)] + ['2' for i in range(1000)])
+        ggplot(data, aes('x', 'y', color='cond')) + \\
+            geom_point(sampling=sampling_random_stratified(50, min_subsample=10))
```

Comment:
It is a little bit coarse. The following code gives pretty similar effect:

```python
...
n = 1100
x = np.random.normal(0, 1, n)
y = np.random.normal(0, 1, n)
cond = np.random.choice(['1', '2'], n, p=[.1, .9])
ggplot({'x': x, 'y': y, 'cond': cond}, aes('x', 'y', color='cond')) + \
    geom_point(sampling=sampling_random_stratified(50, min_subsample=10))
```

But here it is better to use `n = 1000` and `['a', 'b']` instead of `['1', '2']`.